stellargraph/data/explorer.py

```diff
@@ -203,7 +203,7 @@ def _walk(self, rs, start_node, length):
         current_node = start_node
         for _ in range(length - 1):
             neighbours = self.neighbors(current_node)
-            if not neighbours:
+            if not len(neighbours):
```

Comment:
Ah, this is a now-unnecessary change that was required because `neighbours` had switched to a numpy array; see: https://github.com/stellargraph/stellargraph/pull/719#discussion_r371607710